_includes/sites.html

```diff
@@ -0,0 +1,145 @@
+<h1 id="sites-using-cylc">Sites Using Cylc</h1>
+
+<p>Major sites known to be using Cylc are shown on the world map above. <a href="https://www.metoffice.gov.uk/research/approach/collaboration/unified-model/partnership">UM
+Partner</a>
+sites are green; the main Cylc developer sites are red (note these are also
+in the UM Partnership), and others are yellow.</p>
+
+<ul>
+  <li>Africa:
+    <ul>
+      <li>South Africa:
+        <ul>
+          <li><a href="http://www.weathersa.co.za/">South African Weather Service (SAWS)</a></li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+  <li>Asia:
+    <ul>
+      <li>India:
+        <ul>
+          <li><a href="http://www.ncmrwf.gov.in/">National Centre for Medium Range Weather
+Forecasting</a></li>
+        </ul>
+      </li>
+      <li>Korea:
+        <ul>
+          <li><a href="http://web.kma.go.kr/">Korean Meteorological Administration (KMA)</a></li>
+        </ul>
+      </li>
+      <li>Philippines:
+        <ul>
+          <li><a href="http://bagong.pagasa.dost.gov.ph">Philippine Atmospheric, Geophysical, and Astronomical Services
+ Administration (PAGASA)</a></li>
+        </ul>
+      </li>
+      <li>Singapore:
+        <ul>
+          <li><a href="http://www.weather.gov.sg/home/">Meteorological Service Singapore (MSS)</a></li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+  <li>Europe:
+    <ul>
+      <li><a href="https://verc.enes.org/esiwace/services/sup_cylc">Centre of Excellence in Simulation of Weather and Climate in
+ Europe</a></li>
+      <li>Germany:
+        <ul>
+          <li><a href="https://www.dkrz.de/">Deutches Klimarechenzentrum (DKRZ)</a></li>
+          <li><a href="http://www.mpimet.mpg.de/mpimet-startseite/">Max-Planck-Institut für
+Meteorologie</a></li>
+        </ul>
+      </li>
+      <li>Italy:
+        <ul>
+          <li><a href="https://www.cmcc.it">Euro-Mediterranean Center on Climate Change (CMCC)</a></li>
+        </ul>
+      </li>
+      <li>Netherlands:
+        <ul>
+          <li><a href="http://forecast.ewatercycle.org/">eWaterCycle</a> at the Netherlands
+eScience Center, University of Utrecht, TUDelft</li>
+        </ul>
+      </li>
+      <li>Spain:
+        <ul>
+          <li><a href="https://www.bsc.es">Barcelona Supercomputing Center (BSC)</a></li>
+        </ul>
+      </li>
+      <li>UK:
+        <ul>
+          <li><a href="http://www.ceda.ac.uk">Centre for Environmental Data Analysis (CEDA)</a></li>
+          <li><a href="http://www.metoffice.gov.uk">Met Office</a>
+            <div class="testimonial">
+              <p>Cylc has been used at the Met Office since 2012 and operationally since 2014.</p>
+							<p><em>“Every week we have 300+ users who rely on Cylc. At any one
```

Comment:
Tabbing? Also in NIWA testomonial.

Reply:
Oops, a result of not using my usual platform (and editor config)!